### PR TITLE
Build container images

### PR DESCRIPTION
**Pull request by kmadel**
This is a cross post from https://kurtmadel.com/posts/native-kubernetes-continuous-delivery/building-container-images-with-kubernetes/ 

And tests the canonicalUrl metadata that was added a few weeks ago. I encourage everyone to cross post from your own blog sites as well!

### CODE REVIEW

**Comment by kmadel** [comment]
Test

File: content/posts/building-container-images-with-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+#### Basic Configuration for Kaniko with Pod Security Policies
+The following configuration assumes that you have a K8 cluster up and running, and that you have access to `kubectl` to add and modify K8s resources.
+
+1. **Container registry**: I recommend having a staging or sandbox container registry that Kaniko pushes images to, and a production registry that Kaniko does not have access to push to. A separately managed and secured CD job should be used to promote container images from staging to production once required tests/scans/policies have been successfully run against that container image. But of course the Kaniko container image itself (and any other container images used by CD jobs) should always be pulled from the production container registry.
+2. `PodSecurityPolicy`: Once the PodSecurityPolicy admission controller is enabled you will need at least 2 PodSecurityPolicies (acutally you will want to defined and apply all of your Pod Security Policies before enabling the admission controller, not doing so will prevent any pods from being created in the cluster):
```
Comment (logandonley) [inline content/posts/building-container-images-with-kubernetes.md]: Typo on `(acutally`

Comment (kmadel) [reply content/posts/building-container-images-with-kubernetes.md]: I guess Spell Right missed that one, thanks!

**Comment by kmadel** [comment]
@cvega You mind giving this a quick review? Thanks.

**Pull request merged**